src/renderer/components/GroupBase.js

```diff
@@ -79,12 +84,34 @@ class GroupBase extends React.Component {
     this.setState({ image: '' })
   }
 
+  onShowQrVerifyCode () {
+    this.setState({
+      qrCode: ipcRenderer.sendSync('dispatchSync', 'getQrCode')
+    })
+  }
+
+  onShowQrInviteCode () {
+    const { chatId } = this.state
+    this.setState({
+      qrCode: ipcRenderer.sendSync('dispatchSync', 'getQrCode', chatId)
+    })
+  }
```

Comment:
**Note** There are _two_ different QR codes. One is for adding verified contacts, which is done by `onShowQrVerifyCode()` and the other is for displaying an invite code for a verified group, which is done by `onShowQrInviteCode()`.